src/pages/Add2/Add2.jsx

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable no-console */
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import FileBase from 'react-file-base64';
+/* import axios from 'axios'; */
```

Comment:
- [ ] Kindly, remove this old piece of commented out code in order to improve the repo's cleanliness. 👍